src/createServer.js

```diff
@@ -1,3 +1,60 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const url = require('url');
+const { convertToCase } = require('./convertToCase');
+const casesToConvert = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const normalizedUrl = new url.URL(req.url, `http://${req.headers.host}`);
+    const textToConvert = normalizedUrl.pathname.slice(1);
+    const toCase = normalizedUrl.searchParams.get('toCase');
+
+    const errors = [];
+
+    if (!textToConvert) {
+      errors.push({
+        message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+      });
+    }
+
+    if (!toCase) {
+      errors.push({
+        message: `"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+      });
+    } else if (!casesToConvert.includes(toCase.toUpperCase())) {
+      errors.push({
+        message: `This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.`,
+      });
+    }
```

Comment:
- Create a function to get errors and move it to a separate file.
- Create an object with error messages
- "Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER." you can see all available cases in the `casesToConvert` variable so use it in the message as well